api/institutions/views.py

```diff
@@ -459,9 +459,17 @@ class InstitutionDepartmentList(InstitutionImpactList):
     def _format_search(self, search):
         results = search.execute()
 
+        institution_id = self.kwargs['institution_id']
+
         if results.aggregations:
             buckets = results.aggregations['date_range']['departments'].buckets
-            department_data = [{'name': bucket['key'], 'number_of_users': bucket['doc_count']} for bucket in buckets]
+            department_data = [
```

Comment:
Did the serializer-only change not work? I'd rather not add a list comprehension to the view if we don't have to.

Reply:
Since there was already some comprehension occurring, I thought it would be better to stick it in there, but I switched it to the new serializer field.